models/general/knowledge/en/description.txt

```diff
@@ -0,0 +1,5 @@
+!image: https://github.com/fossasia/susi_skill_data/blob/master/models/general/knowledge/en/description.png
+!example: get started
+!description: get started message shown in messenger bots
```

Comment:
Description should be improved in the same PR. Please avoid mini PRs. Provide a proper description take inspiration on skill descriptions on Amazon Alexa.

Provide a proper example. I don't understand what this skill does. Is this a sentence? "get started"